translate/storage/poxliff.py

```diff
@@ -150,8 +151,9 @@ def settarget(self, text, lang='xx', append=False):
 
     def gettarget(self):
         if self.hasplural():
-            strings = [unit.target for unit in self.units]
+            strings = [data.forceunicode(unit.target) for unit in self.units]
             if strings:
+                print(strings)
```

Comment:
Nit: unnecessary print.

Reply:
Apparently I forgot some of the debugging code...